pkg/manager/manifests/permission/role.yaml

```diff
@@ -10,3 +10,9 @@ rules:
   - apiGroups: ["addon.open-cluster-management.io"]
     resources: ["managedclusteraddons"]
     verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: ["coordination.k8s.io"]
+    resources: ["leases"]
+    verbs: ["*"]
+  - apiGroups: ["apps"]
```

Comment:
The agent should not operate deployment on the hub side.

Reply:
Fixed.